src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee-edit-form/ubs-admin-employee-edit-form.component.ts

```diff
@@ -147,10 +147,12 @@ export class UbsAdminEmployeeEditFormComponent implements OnInit, OnDestroy {
       this.tariffs = tariffs;
       if (this.editMode) {
         this.tariffsFromEditForm = this.tariffsFromEditForm.map((editItem) => editItem.id);
-        this.filteredTariffs = this.tariffs.map((tariffItem) => ({
-          ...tariffItem,
-          selected: this.tariffsFromEditForm.includes(tariffItem.id) ? (tariffItem.selected = true) : tariffItem.selected
-        }));
+        this.filteredTariffs = this.tariffs.map((tariffItem) => {
+          return {
+            ...tariffItem,
+            selected: this.tariffsFromEditForm.includes(tariffItem.id) ? true : false
```

Comment:
selected: this.tariffsFromEditForm.includes(tariffItem.id)

Reply:
changed according to comment